### DISCUSSION TITLE
AI’s threat to Google is more about advertising income than being the number one search engine

### DISCUSSION THREAD

**Discussion by Aaron25564448** [1101-GS4719-C/Python-Community-GS4719-C#1150] [General]
https://theconversation.com/ais-threat-to-google-is-more-about-advertising-income-than-being-the-number-one-search-engine-200094
經過這篇文章一提，我才意識到就某方面來說ChatGPT也是一種搜尋引擎，但卻更加有互動性，也更受眾人喜愛。Google未來有可能跌落神壇的原因不是因為搜尋引擎，反而是在搜尋引擎上的廣告投放行銷策略。隨著AI越來越熱門，當這類AI也開始透過廣告投放賺取營收並有所成績的時候，不只是科技，大概也是行銷新時代的來臨。

**Comment by brian431** [comment]
感覺Google在這波浪潮下真的要有很大的改變，畢竟搜尋引擎的廣告收益佔了他們營收很大一部分，如果大家減少使用搜尋引擎的次數，想必對他們會是很大的衝擊。
